distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -195,8 +195,7 @@ public void testFtpPut() throws Exception {
 
         // verify FTP PUT resulted in ingest, catalogued data
         Response response = executeOpenSearch("xml", "q=*", "count=100");
-        response.then()
-                .body("metacards.metacard.size()", equalTo(1));
+        verifyResponse(response, 1);
 
         // clean up test data
         TestCatalog.deleteMetacard(getMetacardIdFromResponse(response));
```

Comment:
I know this wasn't changed, but to prevent future problems we may want to delete _all_ metacards in an `@After tearDown()` method to make sure all tests start with a clean catalog.